app/konnect/dev-portal/access-and-approval/sso.md

```diff
@@ -0,0 +1,117 @@
+
+1. Enter the Dev Portal [Redirect URI](/konnect/dev-portal/access/) for the **Redirect URI**. 
+
+1. [Create a client secret](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app?tabs=client-secret) and save the secret value to configure {{site.konnect_short_name}}.
+
+1. [Use the OIDC well-known discovery endpoint](https://learn.microsoft.com/en-us/azure/active-directory-b2c/secure-api-management?tabs=app-reg-ga#get-a-token-issuer-endpoint) to find and save the `issuer` value. 
+    The `issuer` value will be used as the provider URL when configuring SSO in {{site.konnect_short_name}}.
+
+1. [Add a group claim](https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims?tabs=appui#configure-groups-optional-claims). Enable all the group types settings and select **Group ID** for each setting in the token properties type.
+
+1. [Configure an optional claim](https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims?tabs=appui#configure-optional-claims-in-your-application) with **ID** as the token type and **email** as the claim.
+
+{% endnavtab %}
+{% navtab Okta %}
+things
+{% endnavtab %}
+{% navtab Auth0 %}
+Kong offers OIDC support to allow Single-Sign-on for {{site.konnect_short_name}} and the Dev Portal. This guide shows you how to configure Auth0 for Dev Portal SSO.
+
+## Create an application in Auth0
+
+1. Login to [Auth0](https://auth0.com/), and navigate to the management dashboard for your Auth0 tenant.
+
+1. Navigate to the **Applications > Applications** page to and click "Create Application" create a new application.
+
+    * Name the application, making it easy to reference as the app that manages your Konnect Dev Portal SSO integration.
+    * Select **Machine To Machine** as the **Application Type**.
+    * Select **Auth0 Management API** as the Authorized API.
+    * Authorize the application for at least one Permission to continue, e.g. `read:client_grants`. Note: this permission can be revoked later if desired, but is required to create the application.
+
+1. Submit the form to create the application.
+
+## Configure the Auth0 application
+
+1. Locate your portal's callback URL in the Konnect Identity Provider Configuration Settings page. It will have the form `https://<your_portal_domain>/login`
+1. Navigate to the **Settings** tab and enter the value of your portal's callback URL into each of the following fields:
+
+    * **Application Login URI**
+    * **Allowed Callback URLs**
+
+1. Scroll down to the **Advanced Settings** and open the **Grant Types** tab.
+
+    * Check the box for **Authorization Code** and click **Save Changes**
+
+## Configure Login Action in Auth0
+
+   {:.note}
+   > **Important:** This section is required due to the Auth0 API implementation not being inline with the OIDC standard for the value of the `updated_at` token claim.
```

Comment:
Looks like this notice got combined with the next list item since there is no newline between
```suggestion
   > **Important:** This section is required due to the Auth0 API implementation not being inline with the OIDC standard for the value of the `updated_at` token claim.
   
```